vllm/v1/worker/gpu_model_runner.py

```diff
@@ -1907,8 +1916,17 @@ def _dummy_run(
         num_scheduled_tokens = np.array(num_scheduled_tokens_list,
                                         dtype=np.int32)
 
+        # [Bugfix] This lets FA2 to correctly activate the optimized routine
```

Comment:
Please remove [Bugfix] from comment - when the code is merged, it's no longer a bugfix as it's static code right?